pymc/logprob/utils.py

```diff
@@ -294,3 +305,32 @@ def reconsider_logprob(rv: TensorVariable) -> TensorVariable:
     new_node.op = copy(new_node.op)
     new_node.op.__class__ = original_op_type
     return new_node.outputs[node.outputs.index(rv)]
+
+
+def ignore_logprob_multiple_vars(
```

Comment:
Would it be good to add a test for `ignore_logprob_multiple_vars`?

Reply:
The functionality is a core requirement of `stack_logprob`, and it's covered in this test, when it was first introduced:

https://github.com/pymc-devs/pymc/blob/c709abf87967216346d2bb0757334f428e317718/tests/logprob/test_tensor.py#L134-L171

https://github.com/pymc-devs/pymc/pull/6342

This PR just refactored it out, because it proved useful elesewhere #6529 

***

Having said that, we can definitely add a targeted test.